src/gui/widgets/VisualizationWidget.cpp

```diff
@@ -122,7 +122,9 @@ void VisualizationWidget::paintEvent( QPaintEvent * )
 
 	if( m_active && !Engine::getSong()->isExporting() )
 	{
-		float master_output = Engine::mixer()->masterGain();
+		Mixer * mixer = Engine::mixer();
```

Comment:
Might be able to make this `const Mixer *` after changing the const-ness of the other functions. Didn't have time to test.